sdk/highlight-py/pyproject.toml

```diff
@@ -29,7 +29,7 @@ python = "^3.8"
 blinker = { version = "^1.5", optional = true }
 django = { version = "^4.1", optional = true }
 flask = { version = "^2.2", optional = true }
-fastapi = { version = "^0.92", optional = true }
```

Comment:
Not related but wanted to get this fix in (context: https://github.com/highlight/highlight/issues/5812#issuecomment-1665903069)

Reply:
Sounds good! Unrelated to this PR, but curious if the other Python versions should also be `>=`?

Reply:
I think the other ones make sense to use the caret since they have a major release (reaching at least v1). Don't know much about fast api but it does seem like development moves quickly and we will have rely on our users to let us know if they introduce a breaking change.